### ISSUE TITLE
Icon Request

### ISSUE THREAD

**Issue by tuch96** [Donnnno/Arcticons#48]
Hi all,

please find enclosed a new icon request file.

Many thanks for your support!!

Cheers

**Comment by tuch96** [comment]
Perfect and thanks for your support!! I have seen here on github that a new version (1.2.5) is out. But on f-droid still there an older version 1.2.0. How long does it take that the new version comes out on f-droid?

**Comment by Donnnno** [comment]
Oh, that's strange, I haven't noticed that. Usually, it takes a day or two to update, maybe I did something wrong there. Gonna take a look at it, thanks!

**Comment by tuch96** [comment]
Today I have received the new version over f-droid and it works! Perfect and thanks for your support!!

**Comment by Donnnno** [comment]
Yeah, I dunno what took so long, but I'm happy it's up to date now ;)
